include/SearchForm/tpls/headerPopup.tpl

```diff
@@ -38,7 +38,7 @@
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 *}
-<div id="searchDialog" class="modal fade modal-search" tabindex="-1" role="dialog">
+<div id="searchDialog" class="modal fade modal-search" role="dialog">
```

Comment:
This does address this specific issue but equally create an accessibility issue, so we may have to look deeper at this one as it will not be merged, unless this is addressed